retriever/engines/xmlengine.py

```diff
@@ -4,7 +4,7 @@
 from retriever.lib.dummy import DummyConnection
 from retriever.lib.models import Engine
 from retriever.lib.tools import open_fr, open_fw
-from retriever.lib.engine_tools import xml2csv, sort_csv
+from retriever.lib.engine_tools import xml2csv, sort_csv, xmltocsv
```

Comment:
We need to come up with alternative names for these two functions. `xml2csv`, `xmltocsv`. We can talk more about these two